README.md

```diff
@@ -184,6 +184,41 @@ end
 See [eight_bit_color.rb](lib/super_diff/csi/eight_bit_color.rb)
 for the list of available colors.
 
+### Enable or disable the key
+
+You can disable the key by changing the following config (default: true)
```

Comment:
I think it would read better to have a colon at the end of this sentence so it doesn't stop abruptly:

``` suggestion
You can disable the key by changing the following config (default: true):
```